custom_components/tahoma/climate_devices/atlantic_pass_apcdhw.py

```diff
@@ -0,0 +1,152 @@
+import logging
+from typing import List, Optional
+
+from homeassistant.components.climate import (
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+    SUPPORT_PRESET_MODE,
+    SUPPORT_TARGET_TEMPERATURE,
+    ClimateEntity,
+)
+from homeassistant.components.climate.const import (
+    PRESET_COMFORT,
+    PRESET_ECO,
+    PRESET_NONE,
+)
+from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS
+
+from ..coordinator import TahomaDataUpdateCoordinator
+from ..tahoma_device import TahomaDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+COMMAND_REFRESH_ECO_TARGET_DWH_TEMPERATURE = "refreshEcoTargetDHWTemperature"
+COMMAND_REFRESH_COMFORT_TARGET_DWH_TEMPERATURE = "refreshComfortTargetDHWTemperature"
+COMMAND_REFRESH_TARGET_DWH_TEMPERATURE = "refreshTargetDHWTemperature"
+COMMAND_SET_DWH_ON_OFF_STATE = "setDHWOnOffState"
+COMMAND_SET_ECO_TARGET_DWH_TEMPERATURE = "setEcoTargetDHWTemperature"
+COMMAND_SET_COMFORT_TARGET_DWH_TEMPERATURE = "setComfortTargetDHWTemperature"
+COMMAND_SET_PASS_APCDHW_MODE = "setPassAPCDHWMode"
+
+CORE_BOOST_ON_OFF_STATE = "core:BoostOnOffState"
+CORE_COMFORT_TARGET_DWH_TEMPERATURE_STATE = "core:ComfortTargetDHWTemperatureState"
+CORE_DHW_DEROGATION_AVAILABILITY_STATE = "core:DHWDerogationAvailabilityState"
+CORE_DWH_ON_OFF_STATE = "core:DHWOnOffState"
+CORE_ECO_TARGET_DWH_TEMPERATURE_STATE = "core:EcoTargetDHWTemperatureState"
+CORE_STATUS_STATE = "core:StatusState"
+CORE_TARGET_DWH_TEMPERATURE_STATE = "core:TargetDHWTemperatureState"
+
+DWH_OFF_STATE = "off"
+DHW_ON_STATE = "on"
+
+IO_PASS_APCDHW_CONFIGURATION_STATE = "io:PassAPCDHWConfigurationState"
+IO_PASS_APCDWH_MODE_STATE = "io:PassAPCDHWModeState"
+IO_PASS_APCDHW_PROFILE_STATE = "io:PassAPCDHWProfileState"
+
+PASS_APCDHW_MODE_COMFORT = "comfort"
+PASS_APCDHW_MODE_ECO = "eco"
+PASS_APCDWH_MODE_INTERNAL_SCHEDULING = "internalScheduling"
+PASS_APCDHW_MODE_STOP = "stop"
+
+MAP_HVAC_MODES = {
+    DHW_ON_STATE: HVAC_MODE_HEAT,
+    DWH_OFF_STATE: HVAC_MODE_OFF,
+}
+MAP_REVERSE_HVAC_MODES = dict(map(reversed, MAP_HVAC_MODES.items()))
+
+MAP_PRESET_MODES = {
+    PASS_APCDHW_MODE_COMFORT: PRESET_COMFORT,
+    PASS_APCDHW_MODE_ECO: PRESET_ECO,
+    PASS_APCDWH_MODE_INTERNAL_SCHEDULING: PASS_APCDWH_MODE_INTERNAL_SCHEDULING,
+    PASS_APCDHW_MODE_STOP: PASS_APCDHW_MODE_STOP,
+}
+MAP_REVERSE_PRESET_MODES = dict(map(reversed, MAP_PRESET_MODES.items()))
+
+
+class AtlanticPassAPCDHW(TahomaDevice, ClimateEntity):
+    """Representation of TaHoma IO Atlantic Electrical Heater."""
+
+    @property
+    def temperature_unit(self) -> str:
+        """Return the unit of measurement used by the platform."""
+        return TEMP_CELSIUS
+
+    @property
+    def supported_features(self) -> int:
+        """Return the list of supported features."""
+        return SUPPORT_PRESET_MODE | SUPPORT_TARGET_TEMPERATURE
+
+    @property
+    def min_temp(self) -> float:
+        """Return minimum temperature."""
+        # FIXME: only for naia micro ?
```

Comment:
What is naia micro? 
This comment should be removed as well, but I kept it to be sure that value is reasonable.

Reply:
Naia micro is one of Atlantic device. Documentations says it could produce hot watwer between 30 and 65°.
Maybe another device could produce water more than 65°.

Should i let this limit (30-65) or something more common like 20 - 80 ?